projects/ngx-intl-tel-input/src/lib/ngx-intl-tel-input.component.html

```diff
@@ -12,7 +12,7 @@
         <span class="country-name">{{country.name}}</span>
         <span class="dial-code">+{{country.dialCode}}</span>
       </li>
-      <li class="divider"></li>
+      <li class="divider" *ngIf="preferredCountriesInDropDown.length"></li>
```

Comment:
@Dviejopomata just confirming, Don't we need `strictNullChecker` here? `preferredCountriesInDropDown?.length`

Reply:
Yeah, I thought about it, looked at the code, and the only case it can be null is passing explicitly a  null value.

But now that you say it, why risk it, I added the Elvis operator :)